### ISSUE TITLE
Difference between whisper-bin and whisper-blas-bin ?

### ISSUE THREAD

**Issue by jishnudg** [ggml-org/whisper.cpp#550]
I'm not familiar with the architecture, would appreciate a quick ELI5 explanation of this - - thanks!

**Comment by ggerganov** [comment]
`whisper.cpp` has a built-in matrix-multiplication routine. Additionally, it supports using a 3rd party matrix multiplication provided by the OpenBLAS library. Which one is faster depends on the CPU and hardware that you are running it on.

- `whisper-bin` uses the built-in routine
- `whisper-blas-bin` uses the 3rd party OpenBLAS routine

You can see which one performs better for you
